### ISSUE TITLE
[4.2.2] Cannot stop server safely, need to kill

### ISSUE THREAD

**Issue by Griefed** [Griefed/Survive-Create-Prosper-4#1]
* **Please tell us about your environment:**

   - Survive Create Prosper version: 4.2.2
   - Operating system: Debian GNU/Linux 9 (stretch)
   - Java version: openjdk version "1.8.0_275"
OpenJDK Runtime Environment (build 1.8.0_275-8u275-b01-1~deb9u1-b01)
OpenJDK 64-Bit Server VM (build 25.275-b01, mixed mode)
   
* **Logs:**

None available.

* **What is the current behavior?**

* Stopping the server doesn't actually stop it/shut it down probably. Gets stuck at
```
ThreadedAnvilChunkStorage (undergarden): All chunks are saved
ThreadedAnvilChunkStorage (scp4): All chunks are saved
ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
ThreadedAnvilChunkStorage (DIM1): All chunks are saved
ThreadedAnvilChunkStorage (undergarden): All chunks are saved
```

The newly added mods are not at fault. Disably any one of `Baubley Heart Canisters, MythicBotany, LibX, ProjectMMO` did not solve the issue.
Most current forge version is not at fault either. Downgrading to previous forge versions achieved nothing.
Must be any of the mods updated with SCP Update 4.2.2...but which one?

Step 1: Start the server
Step 2: Wait for it to boot up completely
Step 3: Stop the server
Step 4: ???
Step 5: Observe it not shutting down properly

* **What is the expected behavior?**

The server should shut down properly when issuing the stop command.

**Comment by Griefed** [comment]
kubejs-1604.3.4.134-forge.jar is the culprit.

**Comment by Griefed** [comment]
Fixed in 4.3.3
